### DISCUSSION TITLE
Welcome to Ambie Discussions!

### DISCUSSION THREAD

**Discussion by dpaulino** [jenius-apps/ambie#83] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dcapn8** [comment]
I just want to offer a suggestion for Ambie White Noise:
My first suggestion is the sound that is created near the bow of a small to medium sailboat while sailing in a nice breeze.
   I do no know how to offer this suggestion, but I a am very sure that it would be a very appreciated/used "Noise".  Mike Griffin

**Reply by dpaulino** [reply]
Thanks for the suggestion, we'll look into it!
